geometry_msgs/PoseStamped grasp_pose
---
string success
